instructure/canvas-ios
# Show error if self registration is not allowed

Also removes QR feature flags.

refs: MBL-14722
affects: parent
release note: Added a message for parents that try to create an account using the QR code pairing feature when self registration is not allowed

test plan:
1. On web, go to account settings > Authentication
2. Set `Self registration` to `All account types`
3. Parent app > QR Login > I don't have an account > Scan student QR code
4. You should get the self registration form
5. Set `Self registration` to `Observer accounts only`
6. Repeat step 3
7. You should get the self registration form
8. Set `Self registration` to `Disabled`
9. Repeat step 3
10. You should get an error alert
